### PR TITLE
Fix #529

### LINKED ISSUE #529: Bug - Vulnerable URL is empty
On **Scan History** page, a null value is displayed in **Vulnerable URL** column.
Perhaps a different version of the Nuclei is used, as a result, http_url is not written to the Vulnerability model.

### PR DESCRIPTION
**Pull request by radaram**
Nuclei returns the response to stdout:
`{"template-id":"tech-detect","info":{"name":"Wappalyzer Technology Detection","author":["hakluke"],"tags":["tech"],"reference":null,"severity":"info"},"matcher-name":"nginx","type":"http","host":"https://example.com:443","matched-at":"https://example.com:443","timestamp":"2021-10-31T09:39:47.1571248Z","curl-command":"curl -X 'GET' -d '' -H 'Accept: */*' -H 'Accept-Language: en' -H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/36.0.1944.0 Safari/537.36' 'https://example.com'"}`

It needs to read host_url from matched-at, not from matched.

### CODE REVIEW

**Review by yogeshojha — APPROVED** [review]

**Comment by yogeshojha** [comment]
Hi @radaram This is amazing! Thank you for the quick fix. Thank you for your contributions <3

**Review by yogeshojha — APPROVED** [review]

File: web/reNgine/tasks.py [hunk 1]
```diff
@@ -1337,6 +1337,8 @@ def vulnerability_scan(
                             vulnerability.reference = json_st['info']['reference']
                         if 'matched' in json_st:
                             vulnerability.http_url = json_st['matched']
```
Comment (yogeshojha) [inline web/reNgine/tasks.py:1339]: But we can remove this, not sure if this `matched` is returned to some specific version.

Comment (yogeshojha) [reply web/reNgine/tasks.py:1339]: Can you please confirm, if it is always returning `matched-at` on the latest Nuclei release? Thanks

Comment (radaram) [reply web/reNgine/tasks.py:1339]: Yes, of course, In version nuclei 2.5.2 passed key matched https://github.com/projectdiscovery/nuclei/blob/v2.5.2/v2/pkg/output/output.go#L78
`Matched string `json:"matched,omitempty"`` 

In nuclei 2.5.3(last version) passed key matched-at
`Matched string `json:"matched-at,omitempty"`` https://github.com/projectdiscovery/nuclei/blob/v2.5.3/v2/pkg/output/output.go#L78

Comment (radaram) [reply web/reNgine/tasks.py:1339]: Сan I delete the old condition `if 'matched' in json_st`?

**Comment by yogeshojha** [comment]
Very much appreciated. I looked into it and I think we should keep both. This is because some of them would be running 2.5.2 and would crash. We should keep both conditions for now, until I work on updating the tools features.

I would really appreciate if you could add 

TODO 

section there, maybe something like

# TODO remove in rengine 1.1

**Comment by radaram** [comment]
> I would really appreciate if you could add
> 
> TODO
> 
> section there, maybe something like
> 
> `# TODO remove in rengine 1.1`

Done

**Comment by yogeshojha** [comment]
Thank you very much for this PR. Very much appreciated.

Accept my thanks and thanks for community as well :heart:

**Comment by radaram** [comment]
> Thank you very much for this PR. Very much appreciated.
> 
> Accept my thanks and thanks for community as well ❤️

I am very glad that I was useful :)

**Pull request merged**